docs/en/02_Developer_Guides/07_Debugging/01_Error_Handling.md

```diff
@@ -195,7 +195,7 @@ Only:
 SilverStripe\Core\Injector\Injector:
   Psr\Log\LoggerInterface.errorhandler:
     calls:
-      pushDisplayErrorHandler: [ pushHandler, [ %$DisplayErrorHandler ]] 
+      pushMyDisplayErrorHandler: [ pushHandler, [ %$DisplayErrorHandler ]]
```

Comment:
Would you mind adding single quotes to the service definition? It's a requirement which I should've added earlier, and unrelated to this change, but if you don't mind while you're there that'd be great.

```suggestion
      pushMyDisplayErrorHandler: [ pushHandler, [ '%$DisplayErrorHandler' ]]
```

Same with the other examples in this